### DISCUSSION TITLE
What icons are valid in react-native-tab-view?

### DISCUSSION THREAD

**Discussion by coolsoftwaretyler** [react-navigation/react-navigation#12311] [Q&A] [answered]
Hey folks, the interface for `Route` is typed here:

https://github.com/react-navigation/react-navigation/blob/29813807ea6723e6bc4536fd0906fa637954eb35/packages/react-native-tab-view/src/types.tsx#L32-L39

Where `icon` is just a `string`. But it's unclear to me from any of the documentation what strings might be valid icons or not.

I see MaterialCommunityIcons and Ionicons referenced in some of the example code:

1. https://github.com/react-navigation/react-navigation/blob/29813807ea6723e6bc4536fd0906fa637954eb35/example/src/Screens/BottomTabs.tsx#L38
2. https://github.com/react-navigation/react-navigation/blob/29813807ea6723e6bc4536fd0906fa637954eb35/example/src/Screens/TabView/CustomIndicator.tsx#L20

But icon names from those libraries don't seem to take effect when I write routes like:

```ts
const routes: Route[] = [
  { key: "first", title: "First", icon: "account" },
  { key: "second", title: "Second", icon: "arrow-forward-outline" },
  { key: "third", title: "Third", icon: "ab-testing" },
]
```

**Comment by satya164** [comment, marked as answer]
You can put anything there. You need to render the appropriate icon using the `icon` option which takes a function that returns a react element

**Reply by coolsoftwaretyler** [reply]
Ah gotcha. Thank you
